academy/4-my-own-chain/play-game.md

```diff
@@ -174,8 +175,116 @@ Take the following steps to replace the `TODO`:
 
 That is all there is to it: good preparation and the use of Starport.
 
+## Interact via the CLI
+
+With one game in storage and the game waiting for Bob's move, can Alice make a move? Take a look at the `play-move` message and which parameters it accepts:
+
+```sh
+$ checkersd tx checkers play-move --help
+```
+
+Which returns:
+
+```
+Broadcast message playMove
+
+Usage:
+  checkersd tx checkers play-move [idValue] [fromX] [fromY] [toX] [toY] [flags]
+...
+```
+
+So `Alice` tries:
+
+```sh
+$ checkersd tx checkers play-move 0 0 5 1 4 --from $alice
+                                  ^ ^ ^ ^ ^
+                                  | | | | To Y
+                                  | | | To X
+                                  | | From Y
+                                  | From X
+                                  Game id
+```
+
+Which includes:
+
+```
+...
+raw_log: 'failed to execute message; message index: 0: player tried to play out of
+                                       turn'
+...
+txhash: D10BB8A706870F65F19E4DF48FB870E4B7D55AF4232AE0F6897C23466FF7871B
+```
+
+<HighlightBox type="tip">
+
+If you did not get this `raw_log`, it might be because your transaction was sent asynchronously. You can always query a transaction by using the `txhash` with the following command:
+
+```sh
+$ checkersd query tx D10BB8A706870F65F19E4DF48FB870E4B7D55AF4232AE0F6897C23466FF7871B
+```
+
+And you are back on track:
+
+```
+...
+raw_log: 'failed to execute message; message index: 0: player tried to play out of
+  turn'
+```
+
+</HighlightBox>
+
+Can Bob, who plays _black_, make a move? Can he make a wrong move? For instance a move from `0-1` to `1-0`, which in turn is occupied by one of his pieces?
+
+```sh
+$ checkersd tx checkers play-move 0 1 0 0 1 --from $bob
+```
+
+The computer says no:
```

Comment:
Originally it was "Computer says no", i.e. tongue in cheek. But I don't know how much [this reference](https://www.youtube.com/watch?v=0n_Ty_72Qds) is known.

Reply:
XD I fear that this may be too "niche" a reference for an international audience not to consider the original a grammar fail. But this version is close enough to bridge good grammar and a little easter egg.

Reply:
Ok.